production/src/test/java/io/jaegertracing/kubernetes/ProductionETest.java

```diff
@@ -14,15 +14,9 @@
 package io.jaegertracing.kubernetes;
 
 import io.jaegertracing.kubernetes.deployment.BaseETest;
-import java.io.IOException;
-import org.junit.Ignore;
 
 /**
  * @author Pavol Loffay
  */
 public class ProductionETest extends BaseETest {
-
-  @Ignore("dependency links returns 404 because of old Cassandra image")
```

Comment:
This test will still fail even with fixed c*.

I would like to see added a test which would ping dependencies REST endpoint.